common/test/acceptance/pages/lms/teams.py

```diff
@@ -75,11 +77,15 @@ def verify_my_team_count(self, expected_count):
 
     def click_all_topics(self):
         """ Click on the "All Topics" breadcrumb """
-        self.q(css='a.nav-item').filter(text='All Topics')[0].click()
+        element_selector = 'a.nav-item'
+        self.wait_for_element_visibility(element_selector, "Waiting for topics breadcrumbs to be visible")
```

Comment:
While this doesn't hurt anything, it isn't actually necessary for elements that are always present (not added due to an ajax request following an update). The breadcrumb ("All Topics" and clicking a single topic below) is always present.